src/functionBased/components/TodoItem.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable react/prop-types */
+import React, { useState } from 'react';
+import styles from './TodoItem.module.css';
+
+const TodoItem = (props) => {
+  const [editing, setEditing] = useState(false);
+
+  const handleEditing = () => {
+    setEditing(true);
+  };
+
+  const handleUpdatedDone = (event) => {
+    if (event.key === 'Enter') {
+      setEditing(false);
+    }
+  };
+
+  const completedStyle = {
+    fontStyle: 'italic',
+    color: '#595959',
+    opacity: 0.4,
+    textDecoration: 'line-through',
+  };
+
+  // eslint-disable-next-line react/destructuring-assignment
+  const { completed, id, title } = props.todo;
+
+  const viewMode = {};
+  const editMode = {};
+
+  if (editing) {
+    viewMode.display = 'none';
+  } else {
+    editMode.display = 'none';
+  }
+
+  // useEffect(() => () => {
+  //   console.log('cleaning up...');
+  // }, []);
+
```

Comment:
- [ ] Kindly remove the commented code to increase the code readability.

Reply:
Hello, @safafa thank you for your valuable comment. I already deleted the commented code.